manta-benchmark/tests/web.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019-2022 Manta Network.
+// This file is part of manta-rs.
+//
+// manta-rs is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// manta-rs is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with manta-rs.  If not, see <http://www.gnu.org/licenses/>.
+
+use manta_benchmark::{prove_mint, prove_private_transfer, prove_reclaim, Context};
+use wasm_bindgen_test::{wasm_bindgen_test, wasm_bindgen_test_configure};
+use web_sys::console;
+wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
```suggestion
wasm_bindgen_test_configure!(run_in_browser);

```

Reply:
why should we add this empty line? There has already been an empty line after Line 20

Reply:
sorry the edit didn't work like I wanted, I meant to add a line above it. It's a macro invocation not an import so there should be a newline between them